javav2/usecases/monitor_dynamodb/README.md

```diff
@@ -0,0 +1,640 @@
+# Monitor performance of a DynamoDB application
+
+## Overview
+|                 |                                                                                                                                        |
+|-----------------|----------------------------------------------------------------------------------------------------------------------------------------|
+| Description     | This tutorial shows users how to configure an application's use of DynamoDB to monitor performance by using the AWS SDK for Java (v2). |
+| Audience        | Developer (beginner)                                                                                                                   |
+| Required skills | Java, Maven                                                                                                                            |
+
+## Purpose
+
+In an application that uses DynamoDB, it can be important to understand and monitor DynamoDB’s contributions to
+the application's performance. This tutorial discusses the following ways to monitor performance.
+* Configure client-side performance metrics.
+* View metrics from DynamoDB and from the SDK.
+* Set up alarms on those metrics.
+* Set up Cloudwatch Contributor Insights to dive into specific issues with usage patterns. 
```

Comment:
SHould it be Amazon CloudWatch?

Reply:
No it's actually [`CloudwWatch contributor insights for DynamoDB`](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/contributorinsights.html). I updated to the full, official name. and added 'Amazon CloudWatch` to the 2nd bulleted point.

Reply:
Sounds good